src/components/ChatComponent/NamebarTop.js

```diff
@@ -0,0 +1,45 @@
+import contact from "../../assets/Contact.png";
+import styled from "styled-components";
+
+const StyledNamebarTop = styled.div`
+  width: 100%;
+  height: 18vh; 
+  display: flex;
+  flex-direction: row ;
+  flex-grow: 0 1 0;
+  gap: 2em;
+  padding-left: 40px;
+  justify-self: right;
+  border: solid 2px;
+  border-color: #C6C6C6;
+  fill: #FFFFFF;
+  align-items: center;
+  justify-content: left;
+  font-size: large;
+  font-weight: 700;
+  background-color: white;
+  font-family: Arial, Helvetica, sans-serif
+  margin-left: auto;
+`;
+
+const StyledImage = styled.img`
+  max-height: 80%; /* Limit the height to 80% of the parent container */
+  height: auto; /* Maintain aspect ratio */
+  width: auto; /* Maintain aspect ratio */
+  object-fit: contain; /* Ensure the image scales properly */
```

Comment:
you might wanna try cover instead of contain, unless you've ensured elsewhere the original aspect ratio of the image is correct. This implementation might give you some funny spacing. Try adding a border and test with different img sizes